src/program.js

```diff
@@ -121,7 +129,9 @@ export class Program {
     argv.customPrefs = argv.pref;
 
     const runCommand = this.commands[cmd];
-
+    const configFileName = argv.config;
+    const configObject = parseConfig(argv, configFileName);
+    const newArgv = applyConfigToArgv(argv, configObject, this.defaultValues);
```

Comment:
I filed this to add this code back into `program.js` https://github.com/mozilla/web-ext/issues/775